features-json/webxr.json

```diff
@@ -0,0 +1,368 @@
+{
+  "title":"WebXR Device API",
+  "description":"API for accessing virtual reality (VR) and augmented reality (AR) devices, including sensors and head-mounted displays.",
+  "spec":"https://immersive-web.github.io/webxr/",
+  "status":"other",
+  "links":[
+    {
+      "url":"https://developer.mozilla.org/docs/Web/API/WebXR_Device_API",
+      "title":"MDN Web Docs - WebXR Device API"
+    }
+  ],
+  "bugs":[
+    {
+      "description":"WebXR is in development and is still changing. WebXR was formally developed as WebVR. Because of this there could be bugs inside the standard of the W3C."
```

Comment:
>WebXR was formally developed as WebVR.

There's some historical truth to this. The spec has evolved so much in more than two years that I'm not sure it's relevant to users of the API. Neither Google nor MDN feel the need to mention this in WebXR documentation. 

>Because of this there could be bugs inside the standard of the W3C.

What does this mean?